### PR TITLE
Glossary-to-docs linking [Fixed #1717]

### LINKED ISSUE #1717: Add links to ethereum.org pages in glossary
**Is your feature request related to a problem? Please describe.**

The ethereum.org glossary references some key concepts that we also have pages for. For example "accounts". Where these pages exist we should provide links to them from the glossary so a user can get more information on a chosen topic.

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Added as many links as reasonably possible to other parts of ethereum.org website from terms in glossary (Most of these were to the developer docs pages for continued learning)
- Utilized 👉 emoji on links to other parts of ethereum.org website (differentiating from internal glossary # links, and external links)
- Standardized text for these links to "More about ____" for simplicity and cleaner UI/UX on mobile
- Also added term "token standard" and "gwei" with appropriate linking (ref: #1718)
- Refactored superscript text used in currency definition exponents. Removed unicode superscript, replaced with `<sup></sup>` for markdown interpretation. 

## Related Issue
Fixes #1717
Contributes to #1718

### CODE REVIEW

**Review by ryancreatescopy — APPROVED** [review]

**Comment by ryancreatescopy** [comment]
image

@wackerow I think I said in my review that it might look nice to eventually have even more differentiation between anchor links and links to other pages. This is what I had in mind. Could shortcut this by pulling in the page title and meta description to populate the content. 

Not a blocker to merging this PR, but just to show you what I mean.

**Comment by wackerow** [comment]
@ryancreatescopy Oh that looks great!... I'll take a look at putting that together. I'll open a separate issue to track it so this can be merged/closed in the meantime.

**Review by samajammin — CHANGES_REQUESTED** [review]
Amazing! Thanks for this. 

Only issue: let's update these a tag links to use the markdown link syntax. See my comments in #1725 on the reasoning behind this (it makes it easier for translations).

File: src/content/glossary/index.md [hunk 1]
```diff
@@ -362,10 +406,14 @@ A secret number that allows Ethereum users to prove ownership of an account or c
 
 A method by which a cryptocurrency blockchain protocol aims to achieve distributed [consensus](#consensus). PoS asks users to prove ownership of a certain amount of cryptocurrency (their "stake" in the network) in order to be able to participate in the validation of transactions.
 
+[More about PoS<Emoji text=":point_right:" size={1} ml={`0.75rem`} />](/eth2/#proof-of-stake)
+
 ### proof-of-work (PoW) {#pow}
 
 A piece of data (the proof) that requires significant computation to find. In Ethereum, [miners](#miner) must find a numeric solution to the [Ethash](#ethash) algorithm that meets a network-wide [difficulty](#difficulty) target.
 
+[More about PoW<Emoji text=":point_right:" size={1} ml={`0.75rem`} />](/developers/docs/blocks/#proof-of-work-protocol)
```
Comment (samajammin) [inline src/content/glossary/index.md:415]:
```suggestion
[More about PoW<Emoji text=":point_right:" size={1} ml={`0.75rem`} />](/developers/docs/consensus-mechanisms/pow/)
```

File: src/content/glossary/index.md [hunk 2]
```diff
@@ -362,10 +406,14 @@ A secret number that allows Ethereum users to prove ownership of an account or c
 
 A method by which a cryptocurrency blockchain protocol aims to achieve distributed [consensus](#consensus). PoS asks users to prove ownership of a certain amount of cryptocurrency (their "stake" in the network) in order to be able to participate in the validation of transactions.
 
+[More about PoS<Emoji text=":point_right:" size={1} ml={`0.75rem`} />](/eth2/#proof-of-stake)
```
Comment (samajammin) [inline src/content/glossary/index.md:409]:
```suggestion
[More about PoS<Emoji text=":point_right:" size={1} ml={`0.75rem`} />](/developers/docs/consensus-mechanisms/pos/)
```

File: src/content/glossary/index.md [hunk 3]
```diff
@@ -316,7 +352,9 @@ The third development stage of Ethereum, launched in October 2017.
 
 ### miner {#miner}
 
-A network node that finds valid [proof-of-work](#pow) for new blocks, by repeated pass hashing (see [Ethash](#ethash)).
+A network [node](#node) that finds valid [proof-of-work](#pow) for new blocks, by repeated pass hashing (see [Ethash](#ethash)).
+
+[More about mining<Emoji text=":point_right:" size={1} ml={`0.75rem`} />](/developers/docs/mining/)
```
Comment (samajammin) [inline src/content/glossary/index.md:357]:
```suggestion
[More about mining<Emoji text=":point_right:" size={1} ml={`0.75rem`} />](/developers/docs/consensus-mechanisms/pow/mining/)
```

**Review by samajammin — APPROVED** [review]
👏

**Review by samajammin — APPROVED** [review]

**Pull request merged**